### PR TITLE
Performance Optimization

### PR DESCRIPTION
**Pull request by jzding**
- Use fasthttp to improve webhook performance
- Fix Log level
- Add local dockerfile
- Add README
- Set cpu 10m/50m as default resource allocation for hw-event-proxy

Signed-off-by: Jack Ding <jacding@redhat.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,133 @@
...
+export NODE_NAME=mynode
+export HW_PLUGIN=true; export HW_EVENT_PORT=9087; export CONSUMER_TYPE=HW
+export MSG_PARSER_PORT=9097; export MSG_PARSER_TIMEOUT=10
+export REDFISH_USERNAME=<redfish username>; export REDFISH_PASSWORD=<redfish password>; export REDFISH_HOSTADDR=<BMC host/ip address>
+export LOG_LEVEL=debug
+```
+
+### Install and run Apache Qpid Dispach Router
+```
+sudo dnf install qpid-dispatch-router
```
Comment (aneeshkp) [inline README.md:26]: we can point to the new installer for QDR deployemnt. We will have difference cases  in that repo readme

File: README.md [hunk 2]
```diff
@@ -1 +1,133 @@
...
+```shell
+export VERSION=latest
+export PROXY_IMG=quay.io/redhat_emp1/hw-event-proxy:${VERSION}
+export SIDECAR_IMG=quay.io/redhat_emp1/cloud-event-proxy:${VERSION}
+export CONSUMER_IMG=quay.io/redhat_emp1/cloud-native-event-consumer:${VERSION}
+```
+
+### Setup AMQ Interconnect
+
+Install the `Red Hat Integration - AMQ Interconnect` operator in a new namespace `<AMQP_NAMESPAVCE>` namespace from the OpenShift Web Console.
```
Comment (aneeshkp) [inline README.md:101]: This is not  valid any more . WE don't have AMQP interconnect operator working in 4.9

Comment (aneeshkp) [reply README.md:101]: <AMQP_NAMESPACE> should be <AMQ_NAMESPACE> or just point to the amq installer repo 
the host path will be amqp://router.router.svc.cluster.local

File: examples/manifests/hw-event-proxy.yaml [hunk 3]
```diff
@@ -50,12 +55,14 @@ spec:
...
+            - name: LOG_LEVEL
+              value: "debug"
         - name: cloud-native-event-sidecar
           image: quay.io/redhat_emp1/cloud-event-proxy:latest
           args:
             - "--metrics-addr=127.0.0.1:9091"
             - "--store-path=/store"
-            - "--transport-host=amqp://amq-interconnect.$(AMQP_NAMESPACE).svc.cluster.local"
+            - "--transport-host=amqp://router.$(AMQP_NAMESPACE).svc.cluster.local"
             - "--api-port=8080"
```
Comment (aneeshkp) [inline examples/manifests/hw-event-proxy.yaml:66]: change this to 8089 or any other value. 8080 is very generic and might cause conflict

File: README.md (hunk 2 above)
Comment (jzding) [reply README.md]: done

File: examples/manifests/hw-event-proxy.yaml (hunk 3 above)
Comment (jzding) [reply examples/manifests/hw-event-proxy.yaml]: done

**Review by aneeshkp — APPROVED** [review]

File: hw-event-proxy/go.mod [hunk 4]
```diff
@@ -6,6 +6,7 @@ require (
 	github.com/json-iterator/go v1.1.11
 	github.com/redhat-cne/sdk-go v0.1.1-0.20210723134838-a808941830c9
```
Comment (aneeshkp) [inline hw-event-proxy/go.mod:7]: you  might want to run  go get github.com/redhat-cne/sdk-go@main

File: README.md (hunk 1 above)
Comment (jzding) [reply README.md]: done

File: hw-event-proxy/go.mod (hunk 4 above)
Comment (jzding) [reply hw-event-proxy/go.mod]: updated

**Pull request merged**